library/packages/src/lib/y2packager/release_notes_fetchers/rpm.rb

```diff
@@ -157,6 +157,8 @@ def release_notes_content(package, prefs)
           package.extract_to(tmpdir)
           file, lang = release_notes_file(tmpdir, prefs)
           file ? [File.read(file), lang] : nil
+        rescue PackageDownloadError, PackageExtractionError => e
+          log.warn "Could not download and extract the package: #{e.inspect}"
```

Comment:
Should we display some error message to the user? I'm not sure if failing silently is a good idea here... :thinking:

Reply:
I am not sure. For instance, if the package is missing from the repo, I am not sure whether the user can do something about it.

Reply:
Finally, I have implemented your suggestion, but in the [yast2-installation](https://github.com/yast/yast-installation/pull/997) module.